### ISSUE TITLE
Suspecting NVDA reporting incorrect info regarding hyperlink in word

### ISSUE THREAD

**Issue by LittleStar-VIP** [nvaccess/nvda#13655]
### Steps to reproduce:
Sorry, not sure what is the appropriate title here, please correct it if necessary.

1. make sure at least the style and color items under nvda document setting are checked
2. open a  word file
3. paste or type the following link into the word : http://www.google.com (actually what the link is, is not the main point here, so probably any hyper link is acceptable)
4. press enter
5. press up arrow to move edit cursor back to the previous line
6. nvda says link    style <POINTER(IUnknown) ptr=0x71fd6cb4 at cbe5df0>  white background  underlined  baseline  align left h  style 內文  bright blue  ttp://www.google.co  style <POINTER(IUnknown) ptr=0x71fd6cb4 at cbe57b0>  m  style 內文 black  not underlined 

Several problems identified here:
Problem 1:
I suspected that what nvda is reporting here <POINTER(IUnknown) ptr=0x71fd6cb4 at cbe5df0> should be the name of the style

Problem 2:
According to the above message, the style and color of the link is different between the first h and the rest of the link.
This is because, after the first hof the link, nvda had stated the new style name and the color attribute.

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
nvda 2021.3.5
#### Windows version:
windows 10 home ed 21h2
#### Name and version of other software in use when reproducing the issue:
Office 365 16.0.15028.20228
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
not tried
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
not tried

**Comment by LittleStar-VIP** [comment]
Sorry, others have already posted the new issue

**Comment by XLTechie** [comment]
Not new, but duplicate of #11427.

However that one doesn't cover the strange color reporting, which I think might 
be a Word problem that should be reported in Microsoft feedback.

**Comment by LittleStar-VIP** [comment]
Hi luke,
I have just reported the problem to Microsoft accessibility desk

**Comment by Brian1Gaff** [comment]
Sometimes certain html links in e-mails, apparently in several clients will 
attempt to launch IE instead of a default browser then suggest the link is 
invalid.I've yet to figure out why this happens sometimes and not others 
aand what the link actually looks like as most they are embedded.
  Could just be pad html authoring though, Suppose.
 Brian

***@***.***
Sent via blueyonder.
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----
